upstream-community-operators/falco/falco-operator.v0.5.6.clusterserviceversion.yaml

```diff
@@ -0,0 +1,202 @@
+          "name": "falco-with-traefik-rules"
+        },
+        "spec": {
+          "customRules": {
+            "rules-traefik.yaml": "- macro: traefik_consider_syscalls\n  condition: (evt.num < 0)\n\n- macro: app_traefik\n  condition: container and container.image startswith \"traefik\"\n\n# Restricting listening ports to selected set\n\n- list: traefik_allowed_inbound_ports_tcp\n  items: [443, 80, 8080]\n\n- rule: Unexpected inbound tcp connection traefik\n  desc: Detect inbound traffic to traefik using tcp on a port outside of expected set\n  condition: inbound and evt.rawres >= 0 and not fd.sport in (traefik_allowed_inbound_ports_tcp) and app_traefik\n  output: Inbound network connection to traefik on unexpected port (command=%proc.cmdline pid=%proc.pid connection=%fd.name sport=%fd.sport user=%user.name %container.info image=%container.image)\n  priority: NOTICE\n\n# Restricting spawned processes to selected set\n\n- list: traefik_allowed_processes\n  items: [\"traefik\"]\n\n- rule: Unexpected spawned process traefik\n  desc: Detect a process started in a traefik container outside of an expected set\n  condition: spawned_process and not proc.name in (traefik_allowed_processes) and app_traefik\n  output: Unexpected process spawned in traefik container (command=%proc.cmdline pid=%proc.pid user=%user.name %container.info image=%container.image)\n  priority: NOTICE"
+          }
+        }
+      }]
+spec:
+  apiservicedefinitions: {}
+  customresourcedefinitions:
+    owned:
+    - kind: Falco
+      name: falcos.falco.org
+      displayName: Falco daemonSet
+      description: Represents a Falco running on each node of your cluster.
+      version: v1alpha1
+  description: |-
+    [Falco](https://falco.org) is a behavioral activity monitor designed to detect anomalous activity in your applications. You can use Falco to monitor run-time security of your Kubernetes applications and internal components.
+
+    To know more about Falco have a look at:
+
+    - [Kubernetes security logging with Falco & Fluentd](https://sysdig.com/blog/kubernetes-security-logging-fluentd-falco/)
+    - [Active Kubernetes security with Sysdig Falco, NATS, and kubeless](https://sysdig.com/blog/active-kubernetes-security-falco-nats-kubeless/)
+    - [Detecting cryptojacking with Sysdig’s Falco](https://sysdig.com/blog/detecting-cryptojacking-with-sysdigs-falco/)
+
+    This operator adds Falco to all nodes in your cluster using a DaemonSet. It
+    also provides a Deployment for generating Falco alerts. This is useful for
+    testing purposes.
+
+    ## Settings
+
+    This operator, uses the same options than the
+    [Helm Chart](https://hub.helm.sh/charts/stable/falco), please take a look to
+    all the options in the following table:
+
+    | Parameter                                       | Description                                                          | Default                                                                                |
```

Comment:
@jeff-phillips-18 When using operatorhub.io's preview page (https://operatorhub.io/preview), this markdown renders as a table without borders or dividing lines, and overlaps with the sidebar with links etc. However, on embedded OperatorHub, the table does not render at all, and instead the pipes are visible as plaintext.

Reply:
Default is no borders on tables. We could update operatorhub.io to show borders for tables in the markdown viewer. Thoughts @tlwu2013 ?

For embedded, we currently have table support turned off. I'm not sure why, I don't know if it was intentional or not. @alecmerdler ?

Reply:
@jeff-phillips-18   Yeah, I am with you that we should add some borders and tweak the css styling to the table to make it easier to visually parse the information. Let's work together on that offline, thanks!